src/app/plugins/cat.plugin.ts

```diff
@@ -45,9 +45,7 @@ export class CatPlugin extends Plugin {
 
     if (args[0].includes('breed')) {
       //Simply return the list of supported breeds
-      if (!this._embedBreeds) {
-        await this._getListEmbed();
-      }
+      await this._getListEmbed();
```

Comment:
I think corgo said this on the last pr, this should return the list, and just stick it in the `message.reply`